tests/utests/basic/test_yanglib.c

```diff
@@ -114,7 +114,7 @@ test_yanglib(void **state)
     UTEST_ADD_MODULE(schema_a, LYS_IN_YANG, feats, NULL);
     UTEST_ADD_MODULE(schema_b, LYS_IN_YANG, NULL, NULL);
 
-    assert_int_equal(LY_SUCCESS, ly_ctx_get_yanglib_data(UTEST_LYCTX, &tree));
+    assert_int_equal(LY_SUCCESS, ly_ctx_get_yanglib_data(UTEST_LYCTX, &tree, "%u", ly_ctx_get_change_count(UTEST_LYCTX)));
```

Comment:
please add also some weird format strings, like `"<<%u>>"`, `"%u" , -10`, to make sure that these situations are handled correctly (from the code I believe they are).

Reply:
or just `""`.

Reply:
The implementation should be completely dependent on `vasprintf` functionality so I considered more tests redundant. In any case, the ones you mentioned were added.